common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -293,7 +294,10 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
             VerticalAlignment[] values = VerticalAlignment.values();
             index = (values.length + index) % values.length;
             selectedOverlay.setVerticalAlignmentOverride(values[index]);
-            saveConfigForOverlay();
+            selectedOverlay.getVisibleConfigOptions().stream()
+                    .filter(configHolder -> Objects.equals(configHolder.getFieldName(), "verticalAlignmentOverride"))
```

Comment:
Can't `Overlay#getConfigOptionFromString` be used?

Reply:
Fixed.